web/src/test/java/org/cbioportal/web/GenesetDataControllerTest.java

```diff
@@ -64,46 +64,77 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void fetchGeneticDataItems() throws Exception {
+    public void fetchGeneticDataItemsWithSampleIds() throws Exception {
+        List<GenesetMolecularData> genesetDataItems = createGenesetDataItemsList();
+
+        String geneticProfileId = PROF_ID;
+        List<String> sampleIds = Arrays.asList(SAMPLE_STABLE_ID_1);
+        List<String> genesetIds = Arrays.asList(GENESET_ID_1, GENESET_ID_2);
+
+        // Argument values don't matter, just shows which fetchGenesetData is called
```

Comment:
Why don't they matter? I think they do. We are saying that when genesetDataService gets these args, it produces this value. We are setting up our expectations for this dependency in the context of this unit test.

Reply:
deleted the comment